app/assets/javascripts/discourse/app/controllers/user-activity-bookmarks.js

```diff
@@ -81,13 +66,29 @@ export default Controller.extend({
 
     this.set("loadingMore", true);
 
-    return this.model
-      .loadMore({ q: this.q })
+    return this._loadMoreBookmarks({ q: this.q })
       .then((response) => this._processLoadResponse(response))
       .catch(() => this._bookmarksListDenied())
       .finally(() => this.set("loadingMore", false));
   },
 
+  _loadMoreBookmarks(additionalParams) {
+    if (!this.model.loadMoreUrl) {
+      return Promise.resolve();
+    }
+
+    let moreUrl = this.model.loadMoreUrl;
+    if (additionalParams) {
+      if (moreUrl.includes("?")) {
+        moreUrl += "&" + $.param(additionalParams);
```

Comment:
We are trying to move away from jQuery stuff. I think the easiest way would be to do the same as:

https://github.com/discourse/discourse/blob/cb63c297b34dd793959bc9bf0c8b3f722d158e83/app/assets/javascripts/discourse/app/models/login-method.js#L47-L53

Ideally we should have this kind of thing in a library function to compensate for the loss of jQuery.

Reply:
Looks great, and yeah I wasn't implying we needed that replacement lib done in this PR, I just meant in general it would be good if we had one :)